tools/webpack-jumpstart

```diff
@@ -170,9 +170,9 @@ unpack_from_cache "${tag}"
 
 # "${changed_files}" is empty unless --partial was given
 for file in ${changed_files}; do
-    # don't touch files that have nothing to do with webpack
-    if grep -q -F ${file} dist/*/Makefile.deps; then
-        message TOUCH "${file}"
-        touch "${file}"
-    fi
+    # remove any webpacks that mention this file
+    grep -l "${file}" dist/*/Makefile.deps | cut -f2 -d/ | while read -r webpack; do
```

Comment:
I don't have a strong opinion either way. It feels a tad more robust to remove the whole webpack. I just think this should probably grow a `sort -u` to avoid showing the "REMOVE" message for the same webpack multiple times?

Reply:
I think that shouldn't be possible: The `grep -l` will display each filename at most once, and we won't hit the same file more than once because on the next iteration (to check the next changed file) the `Makefile.deps` that mentioned it will also have disappeared.

But it raises an interesting point: in theory, if all of the webpacks got deleted (like if some babelrc changed, or so) then, for the next file changed after that, our glob would fail to match, and we'd end up with errors about that.

Reply:
I meant that if you change e.g. pkg/systemd/terminal.{jsx,scss} (which is quite common), it would go through that loop twice. But you are right that this concerns the outer loop, not the inner one, so we'd have the duplication either way.

Reply:
The point is that, on the second run through the loop, we run `grep` again.  Since we deleted the entire webpack in the previous iteration, we won't find it on the list of things that need to be deleted, the second time around, therefore no double message.

Reply:
Oh, of course, sorry. Reviewing PRs during meetings, argh :brain: